apisix/admin/v3_adapter.lua

```diff
@@ -122,4 +118,55 @@ function _M.filter(body)
 end
 
 
+local function filter(body, args)
+    if not args.name and not args.label and not args.uri then
+        return
+    end
+
+    for i = #body.list, 1, -1 do
+        local name_matched = true
+        local label_matched = true
+        local uri_matched = true
+        if args.name then
+            local matched = re_find(body.list[i].value.name, args.name, "jo")
+            if not matched then
+                name_matched = false
+            end
+        end
+
+        if args.label then
+            local matched = re_find(body.list[i].value.label, args.label, "jo")
```

Comment:
We should match one of the labels?
https://github.com/apache/apisix/blob/a8a692d9a58a9a0185e22cfebbe2c0f000ba41f3/apisix/schema_def.lua#L693

Reply:
update